freqtrade/freqtradebot.py

```diff
@@ -1049,7 +1049,7 @@ def handle_stoploss_on_exchange(self, trade: Trade) -> bool:
         Check if trade is fulfilled in which case the stoploss
         on exchange should be added immediately if stoploss on exchange
         is enabled.
-        # TODO-lev: liquidation price always on exchange, even without stoploss_on_exchange
+        # TODO: liquidation price always on exchange, even without stoploss_on_exchange
```

Comment:
I have no solution for this, maybe demand that stoploss on exchange be used for futures? Is there any reason to not use it if it's implemented?

Reply:
I'd actually leave this for the moment (as TODO-lev so we don't forget).

if we assume that we set out "liquidation price offset" to 5% above the real liquidation (so we don't actually encounter a liquidation), then freqtrade will also handle this off exchange - _assuming_ we treat that price as stoploss (or rather - stoploss becomes `max(stoploss, liquidationprice)`.

Reply:
> I'd actually leave this for the moment (as TODO-lev so we don't forget).
> 
> if we assume that we set out "liquidation price offset" to 5% above the real liquidation (so we don't actually encounter a liquidation), then freqtrade will also handle this off exchange - _assuming_ we treat that price as stoploss (or rather - stoploss becomes `max(stoploss, liquidationprice)`).

There's the possibility that the price blows through the 5% buffer though, especially if you're using high leverage

Reply:
we have the same possibility for stoploss on exchange ... (we define the stop and the limit (by default 1% below) - which stops can blow through for sure.

That's also "not really" handled for stoploss on exchange either - so i guess keeping it as regular TODO will be fine (i'd however ask you to add the "blow through" comment to the file, so we know what it's about when looking at this months from now).